railties/lib/rails/generators/rails/app/templates/config/puma.rb

```diff
@@ -0,0 +1,9 @@
+threads_count = ENV.fetch("MAX_THREADS") { 5 }.to_i
```

Comment:
Updated to use a prefix `RAILS_MAX_THREADS`. I think that's the last piece of actionable feedback since `--skip-puma` is fine as a flag.